website_user_login_redirect_custom/controllers/main.py

```diff
@@ -0,0 +1,77 @@
+# © 2025 Cetmix OU
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.web.controllers.main import Home
+
+
+class WebsiteRedirectCustom(Home):
+    """
+    Inherit from Home to override login behavior
+    This works even without auth_signup module
+    """
+
+    def _should_redirect_custom(self):
+        """Check if we should perform custom redirect"""
+        if not request.session.uid:
+            return False
+
+        # Do not redirect backend/internal users
+        if request.env.user.has_group("base.group_user"):
+            return False
+
+        enabled = (
+            request.env["ir.config_parameter"]
+            .sudo()
+            .get_param("website_user_login_redirect_custom.enabled", "0")
+        )
+        return enabled == "1"
+
+    def _get_custom_redirect_url(self):
+        """Get custom redirect URL or False if not set/valid"""
+        url = (
+            request.env["ir.config_parameter"]
+            .sudo()
+            .get_param("website_user_login_redirect_custom.url", "/")
+        )
+
+        if not url:
+            return False
+
+        url = url.strip()
+
+        # Validate URL format
+        if (
+            url.startswith("/")
+            and not url.startswith("//")
+            and "://" not in url
+            and "javascript:" not in url.lower()
+        ):
+            return url
+
+        return False
+
+    @http.route("/web/login", type="http", auth="public", website=True, sitemap=False)
+    def web_login(self, redirect=None, **kw):
+        """Override standard login route to apply redirect after login."""
+        response = super().web_login(redirect=redirect, **kw)
+
+        # Only redirect if:
+        # 1. No explicit redirect parameter was provided
+        # 2. User is logged in and is portal/website user
+        # 3. Feature is enabled
+        # 4. We have a valid redirect response
+        # 5. Custom URL is valid and safe
+        if (
+            not redirect
```

Comment:
The idea of this module is to force the redirection to the custom url. Even if there is already a redirect.